tests/Report/junit.xsd

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
next reason to incorporate xsd file in repo.
external specification could be changed. Especially since it's not standard like ECMA-* or sth...